ext/package.json

```diff
@@ -2,14 +2,22 @@
   "name": "ext",
   "version": "1.0.0",
   "dependencies": {
-    "aws-sdk": "2.1061.0",
     "@azure/storage-blob": "12.9.0",
+    "@types/ini": "^4.1.0",
+    "aws-sdk": "2.1061.0",
+    "commander": "^12.1.0",
     "dotenv": "16.0.3",
-    "electron-updater": "^5.3.0"
+    "electron-updater": "^5.3.0",
+    "ini": "^4.1.3",
+    "typescript-eslint": "^7.12.0"
   },
   "devDependencies": {
+    "@electron/notarize": "2.3.2",
     "electron": "30.0.6",
     "electron-builder": "23.6.0",
-    "@electron/notarize": "2.3.2"
+    "eslint": "9.x",
```

Comment:
How do I use eslint on this repo? `npx eslint` at top level didn't do anything. `npx eslint` within the ext directory did stuff, giving a lot of errors. Just checking what expectation is being set here?

Reply:
The errors are expected (for now). There is too much legacy code that does not conform to eslint's default rules. Linting was introduced in this PR to help me check for unsorted imports and missing semicolons. More linting-related changes will be added gradually.